.github/actions/upload/action.yml

```diff
@@ -0,0 +1,59 @@
+name: Upload packages to RWS
```

Comment:
Upload packages by RWS? (to S3).

Reply:
Probably you are right. I will think about it and fix it in the next PR.